### PR TITLE
[QA] give LROs return types in authoring

### PR DESCRIPTION
**Pull request by kristapratico**
Resolves https://github.com/Azure/azure-sdk-for-python/issues/25304

These transforms give LROs return types that were missing in the previous beta release for authoring.

### CODE REVIEW

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md [hunk 1]
```diff
@@ -68,6 +68,64 @@ title: QuestionAnsweringAuthoringClient
...
+      schema: {
+        "$ref": "#/definitions/JobState"
+      }
+    };
+- where-operation: QuestionAnsweringProjects_UpdateQnas
+  transform: |
+    $["x-ms-pageable"] = {
+      "nextLinkName": "nextLink",
+      "itemName": "value"
+    };
```
Comment (kristapratico) [inline sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md:105]: The x-ms-pageable addition should be updated in the swagger. Discussion in https://github.com/Azure/azure-sdk-for-python/issues/25304

**Review by heaths — APPROVED** [review]
The `_operations.py` files are generated, right? Rest looks good. I'll be sure to update my instances of shared transforms. Either I missed some things the first time, or the swagger was updated.

**Comment by kristapratico** [comment]
> The `_operations.py` files are generated, right? Rest looks good. I'll be sure to update my instances of shared transforms. Either I missed some things the first time, or the swagger was updated.

Yep generated. Thanks Heath!

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md [hunk 2]
```diff
@@ -68,6 +68,64 @@ title: QuestionAnsweringAuthoringClient
...
+directive:
+- where-operation: QuestionAnsweringProjects_DeleteProject
+  transform: |
+    $.responses["200"] = {
+      description: "Project delete job status.",
+      schema: {
+        "$ref": "#/definitions/JobState"
+      }
+    };
+- where-operation: QuestionAnsweringProjects_DeployProject
```
Comment (heaths) [inline sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md:84]: I don't think that's correct - at least, not as authored. Maybe as intended. Progression:

1. https://github.com/Azure/azure-rest-api-specs/blob/8ff0e3b8dc12cd793f4f2208d76f9f3a7f51176c/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json#L443
2. https://github.com/Azure/azure-rest-api-specs/blob/8ff0e3b8dc12cd793f4f2208d76f9f3a7f51176c/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json#L488
3. https://github.com/Azure/azure-rest-api-specs/blob/8ff0e3b8dc12cd793f4f2208d76f9f3a7f51176c/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json#L517

Comment (heaths) [reply sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md:84]: Or, were you saying - and I completely misunderstood - that the Operation-Location URI itself is returning a final response that itself points back to the `get` of this and other operations?

Comment (kristapratico) [reply sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md:84]: Not Operation-Location, but our poller impl does an extra GET on the original Uri (used in the PUT) once the operation succeeds. Here's an example in recording:

PUT: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/recordings/test_create_and_deploy_project.pyTestCreateAndDeploytest_deploy_project.json#L209

GET (succeeded): https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/recordings/test_create_and_deploy_project.pyTestCreateAndDeploytest_deploy_project.json#L263

extra GET on the original Uri: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/recordings/test_create_and_deploy_project.pyTestCreateAndDeploytest_deploy_project.json#L292

Comment (heaths) [reply sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md:84]: Thanks. I get it now. After talking with @annelo-msft it seems .NET does this as well. /cc @AlexanderSher if that's incorrect, but I'll update my transforms as well. Thanks for catching this and clarifying, and sorry I missed that initially!

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md [hunk 3]
```diff
@@ -68,6 +68,64 @@ title: QuestionAnsweringAuthoringClient
...
+      }
+    };
+- where-operation: QuestionAnsweringProjects_UpdateQnas
+  transform: |
+    $["x-ms-pageable"] = {
+      "nextLinkName": "nextLink",
+      "itemName": "value"
+    };
+    $.responses["200"] = {
+      description: "Update QnAs job state.",
```
Comment (heaths) [inline sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md:107]: I would change this to the equivalent of the PUT or GET final model:
> All the QnAs of a project.

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md [hunk 4]
```diff
@@ -68,6 +68,64 @@ title: QuestionAnsweringAuthoringClient
...
+      }
+    };
+- where-operation: QuestionAnsweringProjects_UpdateSources
+  transform: |
+    $["x-ms-pageable"] = {
+      "nextLinkName": "nextLink",
+      "itemName": "value"
+    };
+    $.responses["200"] = {
+      description: "Update sources job state.",
```
Comment (heaths) [inline sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md:119]: > All the sources of a project.

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md [hunk 5]
```diff
@@ -68,6 +68,64 @@ title: QuestionAnsweringAuthoringClient
...
+    $.responses["200"] = {
+      description: "Project delete job status.",
+      schema: {
+        "$ref": "#/definitions/JobState"
+      }
+    };
+- where-operation: QuestionAnsweringProjects_DeployProject
+  transform: |
+    $.responses["200"] = {
+      description: "Deploy job state.",
```
Comment (heaths) [inline sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md:87]: > Project deployment details.

**Pull request merged**